src/index.css

```diff
@@ -404,10 +408,20 @@ video {
 
 dialog {
   position: fixed;
-  top: 90%;
-  left: 30%;
+  top: 20%;
+  left: 89%;
   transform: translate(-50%, -50%);
   z-index: 1;
   width: fit-content;
   background-color: rgba(0,0,0,0.5);
 }
+
+#ov-status{
+  position: fixed;
+  top: 21%;
+  left: 5%;
+  transform: translate(-80%, -100%);
+  text-align: left;
+}
```

Comment:
probably shouldn't used fixed position for this, we might want to fix the CSS such that we use a horizontal grid display to get the look we are going for especially since we are getting rid of toggle dialog box